### ISSUE TITLE
Debian python3.9 test suite fails

### ISSUE THREAD

**Issue by jeremiah** [vinland-technology/flict#104]
# Description

Test suite 

## Steps for reproduction

Run this command in the flict dir;
`$  python3.9 -m pytest tests`

## Expected behavior
Successful execution of test suite.

## Actual behavior
```
ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
__main__.py: error: unrecognized arguments: --bandit --random-order --random-order-bucket=global 
  inifile: /home/jeremiah/Packaging/clean/flict/setup.cfg
  rootdir: /home/jeremiah/Packaging/clean/flict  
```
## Environment

 | tool    | version    | how to get it       |
 | ------- | ---------- | ------------------- |
 | flict   | 91-move-makefile branch | `flict --version`   |
 | python3 | 3.9.7 | `python3 --version` |
 | os      | Debian testing | n.a.                |

Using flict branch  on Debian testing

## Additional logs

Removing `--random-order --random-order-bucket=global --bandit` fixes the error. This *may* be due to missing python test packages in Debian.

**Comment by hesa** [comment]
@jens-erdmann and @priv-kweihmann ... do you have any ideas? 

Heah yeah yeah,.... it's probably obvious to you. Next beer is on me

**Comment by priv-kweihmann** [comment]
Some of the dependencies from `requirements-dev.txt` are missing.
Such as `pytest-random-order` and `pytest-bandit`.

Don't know if that has been documented anywhere but for all of development (and I consider packaging a part of development :smile: ) one needs to install `pip3 install -r requirements-dev.txt`

**Comment by jeremiah** [comment]
Packaging is really part of integration. `</pedantic>`

See discussion here regarding pip: #106

**Comment by hesa** [comment]
Tests are performed using the `dev` requirements. No need to install these when not testing, as for a normal user.
Will update CONTRIBUTING.md and then close this issue.

**Comment by hesa** [comment]
Included in #123
